src/kudu/consensus/raft_consensus.cc

```diff
@@ -2505,6 +2511,15 @@ const string& RaftConsensus::peer_uuid() const {
   return local_peer_pb_.permanent_uuid();
 }
 
+std::pair<string, unsigned int> RaftConsensus::peer_hostport() const {
+  if (local_peer_pb_.has_last_known_addr()) {
```

Comment:
this seems to be not being protected by lock and not needing one. We can add a comment later.